src/jdk.accessibility/share/classes/com/sun/java/accessibility/util/AWTEventMonitor.java

```diff
@@ -50,166 +50,23 @@
      */
     public AWTEventMonitor() {}
 
-    /**
-     * The current component with keyboard focus.
-     *
-     * @see #getComponentWithFocus
-     *
-     * @deprecated This field is unused; to get the component with focus use the
-     * getComponentWithFocus method.
-     */
-    @Deprecated(since="8", forRemoval=true)
-    static protected Component componentWithFocus = null;
-
     static private Component componentWithFocus_private = null;
```

Comment:
Can we rename the "new" "xx_private" fields to the shorter version? The javadoc for the protected fields can be applied to the "private version"(but w/o @Deprecated tag).

So in fact we can change "protected" to the "private", drop the "@Deprecated" tags, and delete the "xx_private" versions of the fields.

Reply:
I don't think the javadoc is needed since these are now private fields.
I can however rename them to remove _private.